cmd/clusterctl/client/init_test.go

```diff
@@ -776,6 +779,12 @@ spec:
       - name: credentials
         secret:
           secretName: ${SOME_VARIABLE}
+status:
+  conditions:
+  - message: Deployment has minimum availability.
+    reason: MinimumReplicasAvailable
+    status: "True"
+    type: Available
```

Comment:
Yup. I know this looks weird but it will be a whole lot cleaner if the Init tests were also written with envtest.